src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -395,7 +394,7 @@ private static string[] GetParsedAuthValueElements(string wwwAuthenticateValue)
         {
             string[] result;
             char[] charsToTrim = { ',', ' ' };
-            var authValuesSplit = CoreHelpers.SplitWithQuotes(wwwAuthenticateValue, ' ' );
+            List<string> authValuesSplit = CoreHelpers.SplitWithQuotes(wwwAuthenticateValue, ' ');
```

Comment:
think we can use var, we do not have a rule to use explicit only.

Reply:
Personal 2c - this improves readability. But if the team prefers `var`, happy to rollback.

Reply:
being explicit is always good I guess, keep it.